_tools/opengazettes.md

```diff
@@ -1,21 +1,38 @@
 ---
 layout: tool
+name: opengazettes
```

Comment:
Why is this tool different to all the others? I think this is the better way to go about it (although the ``name`` entry is implicit and not needed).